core/src/main/scala/kafka/server/AlterPartitionManager.scala

```diff
@@ -77,25 +76,14 @@ object AlterPartitionManager {
   def apply(
     config: KafkaConfig,
     metadataCache: MetadataCache,
+    alterPartitionChannelManager: BrokerToControllerChannelManager,
```

Comment:
nit: I think `alterPartition` is redundant here. Could we call it `channelManager`?